app/server/models/upload.py

```diff
@@ -0,0 +1,33 @@
+from sqlalchemy.ext.hybrid import hybrid_property
+from server.utils.amazon_s3 import get_file_url
+
+from server import db
+from server.models.utils import ModelBase
+from server.models.user import User
+from server.models.transfer import CreditTransfer
+
+class UploadedImage(ModelBase):
+    __tablename__ = 'uploaded_image'
+
+    filename = db.Column(db.String)
+    image_type = db.Column(db.String)
+    credit_transfer_id = db.Column(db.Integer, db.ForeignKey(CreditTransfer.id))
+    user_id = db.Column(db.Integer, db.ForeignKey(User.id))
+
+    @hybrid_property
+    def image_url(self):
+        return get_file_url(self.filename)
+
+class UploadedDocument(ModelBase):
+    __tablename__               = 'uploaded_document'
+
+    kyc_application_id = db.Column(db.Integer, db.ForeignKey('kyc_application.id'))
```

Comment:
how come here the foreign key is a string whereas in `UploadedImage` it's a class reference? is class reference preferable for any reason? they seem like potential for circular dependencies otherwise

Reply:
No reason. Generally, a string is used if it is referencing a class before it is created in the model file.

`UploadedDocument` and `UploadedImage` should be merged into a single class. Issue here: https://github.com/teamsempo/SempoBlockchain/issues/11